content/collections/experiment-apis/en/experiment-management-api-deployments.md

```diff
@@ -125,10 +125,10 @@ Edit a deployment that experiments or flags can be assigned to.
 
 ### Query parameters
 
-|Name|Description|
-|---|----|
-|`label`| Optional | Deployment's label. Must contain alphanumeric and/or `_`, `-` characters. |
-|`archive`| Optional | string | Soft delete or restore deployment. |
+|Name|Requirement|Type|Description|
+|---|----|----|----|
+|`label`| Optional | string | Deployment's label. Must contain alphanumeric and/or `_`, `-` characters. |
+|`archive`| Optional | boolean | Soft delete or restore deployment. |
```

Comment:
where did you get that `archive` was a string? Does something else need to get updated?

Reply:
Not sure where this is from. It was in the old version of this doc as well.